README.md

```diff
@@ -1 +1,145 @@
-# doctor_appoitment_front_end
\ No newline at end of file
+
+# Doctors Appoinments Frontend
```

Comment:
# Backend re-review

- [ ] This `export` command is not working on my end:

### Your instructions:

![image](https://github.com/zeff96/doctor_appoitment_front_end/assets/57421823/d41fb553-bdd4-4f25-bbdf-412cd9448662)

### The result
#### I run `rails secret`, do the export, check that the env variable exists with the `env` command (it does), then I open a console and try to call for it, but it's `nil`:

![image](https://github.com/zeff96/doctor_appoitment_front_end/assets/57421823/c54a5042-599d-46df-a6ff-3d1aa3141064)

## The solution

Do it the old way opening an editor and adding it as a credential inside Rails.  These are the steps:

```
1. Run `rails secret`.  Copy the key.
2. Remove config/master.key and config/credentials.yml.enc if they exist.
3. Run EDITOR="code --wait" bin/rails credentials:edit
4. In the editor that opens, add this:  devise_jwt_secret_key: <the key you copied in step 1>
5. Save the file and close the editor.  New master.key, credentials.yml.enc files will be generated, and the key will be stored in `Rails.application.credentials.devise_jwt_secret_key`.
```

After performing these steps, you can open a console, call for `Rails.application.credentials.devise_jwt_secret_key`, and the key will be there:

![image](https://github.com/zeff96/doctor_appoitment_front_end/assets/57421823/efccb27f-5342-4016-aaad-4b7794e7cc2b)

Also, the tests will finally pass:

![image](https://github.com/zeff96/doctor_appoitment_front_end/assets/57421823/762878a2-4ccf-439e-a0c6-538cffff791d)

## Conclusion

If the `export` procedure didn't work for me, chances are it won't work for other new users (like other reviewers, the final evaluator, and potential employers).

Therefore, kindly make your instructions reflect this.  The instructions to include on your readme have already been written for you above in this very review comment.  You can just copy them.

That way, your chances with the final evaluator and potential employers, will be vastly improved. 💪